client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -41,8 +41,8 @@ function getHoldMessages( siteSlug, translate ) {
 			supportUrl: 'https://support.wordpress.com/suspended-blogs/',
 		},
 		NON_ADMIN_USER: {
```

Comment:
I'm wondering whether we should create a new case here like `NON_SITE_OWNER_USER` instead of reusing the non_admin_user one. It's not a big deal but we might be introducing some technical dept, especially if non-admin users are treated differently in the future.